scss/_company_home.scss

```diff
@@ -0,0 +1,23 @@
+.company-home {
+  .company {
+    font-size: 1.5em;
+  }
+
+  .user {
+    font-size: 1em;
+  }
+
+  .rc-logo {
+    margin: 3em 0;
+    height: 10em;
+
+    img {
+      height: 100%;
+    }
```

Comment:
Is there a reason to wrap this inside another tag? Can you not just set the height and margin directly on the image?